src/pages/index.js

```diff
@@ -0,0 +1,331 @@
+  const cardDeleteBtn = cardElement.querySelector(".card__trash-button");
+
+  cardLikedBtn.addEventListener("click", () => {
+    if (cardLikedBtn.classList.contains("card__like-button_liked")) {
+      api
+        .nonlikedCard(data._id)
+        .then(() => {
+          cardLikedBtn.classList.remove("card__like-button_liked");
+        })
+        .catch(console.error);
+    } else {
+      api
+        .likeCard(data._id)
+        .then(() => {
+          cardLikedBtn.classList.add("card__like-button_liked");
+        })
+        .catch(console.error);
+    }
+  });
+
+  cardDeleteBtn.addEventListener("click", () =>
+    handleCardDelete(cardElement, data)
+  );
+
+  image.addEventListener("click", () => {
+    openModal(previewModal);
+    preModalCaptionEl.textContent = data.name;
+    preModalImage.src = data.link;
+    preModalImage.alt = data.name;
+  });
+
+  cardNameEl.textContent = data.name;
+  image.src = data.link;
+  image.alt = data.alt;
+
+  if (data.isLiked) {
+    cardLikedBtn.classList.add("card__like-button_liked");
+  }
+  return cardElement;
+}
+
+function handleDeleteSubmit() {
+  // evt.preventDefault()
+  setSubmitButtonText(deleteModalDelBtn, "Deleting");
+
+  api
+    .deleteCard(selectedCardId)
+    .then(() => {
+      selectedCard.remove();
+      closeModal(cardDeleteModal);
+      newCardForm.reset();
+    })
+    .catch(console.error)
+    .finally(() => {
+      setSubmitButtonText(deleteModalDelBtn, "Delete");
+    });
+}
+function handleCardDelete(cardElement, data) {
+  selectedCard = cardElement;
+  selectedCardId = data._id;
+  openModal(cardDeleteModal);
+}
+deleteModalDelBtn.addEventListener("click", handleDeleteSubmit);
+deleteModalCanclBtn.addEventListener("click", () => {
+  closeModal(cardDeleteModal);
+});
+
+function openModal(modal) {
+  modal.classList.add("modal_opened");
+  document.addEventListener("keydown", handleEscKeyPress);
+  document.addEventListener("click", handleOverlay);
+}
+
+function closeModal(modal) {
+  modal.classList.remove("modal_opened");
+  document.removeEventListener("keydown", handleEscKeyPress);
+  document.removeEventListener("click", handleOverlay);
+}
+
+function handleEscKeyPress(evt) {
+  if (evt.key === "Escape") {
+    const openedModal = document.querySelector(".modal_opened");
+    if (openedModal) {
+      closeModal(openedModal);
+    }
+  }
+}
+
+function handleEditFormSubmit(evt) {
+  evt.preventDefault();
+  const profileSubmitBtn = evt.submitter;
+  profileSubmitBtn.textContent = "Saving...";
+
+
+
+  api
+    .editUserInfo({
+      name: editModalProfileName.value,
+      about: editModalProfileDescription.value,
+    })
+    .then((data) => {
+      profileDescriptionElement.textContent = data.about;
+      profileNameElement.textContent = data.name;
+      closeModal(editProfileModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      cardModalSubmitBtn.textContent = "Save";
```

Comment:
`cardModalSubmitBtn`


The profile button is called `profileSubmitBtn` in line 225. Please, use the same variable in `finally` to return the button text back